src/functions.php

```diff
@@ -56,10 +56,6 @@ function uploadImages(array $vcards, $config, $configPhonebook, callable $callba
     $mapFTPUIDtoFTPImageName = [];                      // "9e40f1f9-33df-495d-90fe-3a1e23374762" => "9e40f1f9-33df-495d-90fe-3a1e23374762_190106123906.jpg"
     $timestampPostfix = substr(date("YmdHis"), 2);      // timestamp, e.g., 190106123906
     $configImagepath = $configPhonebook['imagepath'] ?? NULL;
-    if (!$configImagepath) {
```

Comment:
Ich sehe diesen check nicht wirklich als redundant.

Ja, inhaltlich machen die beiden Tests dasselbe. Von daher könnte man sie ggfs in eine eigene funktion auslagern, die dann von den zwei Stellen aufgerufen wird.

Der 1. Test in RunCommand macht Sinn, weil er SEHR FRÜH (vor download der vCards) auf einen späteren Showstopper hinweist. Also => Benutzerfreundlichkeit!

Der 2. Test in functions.php macht Sinn, weil der Code von `uploadImages` innerhalb von functions nicht sicher sein kann, ob der aufrufende Code den 1. Test vorher gemacht hat. Liegt ja außerhalb seiner Kontrolle.
Allerdings muss `uploadImages`es 100% wissen, sonst geht diese Funktion ja nicht. Hier geht es also um Sicherheit/Wasserdichtigkeit, wenn irgendjemand irgendwann mal den 1. Test entfernt.

=> Bitte beide Tests drinlassen. 
Wenn Du es schöner machen willst, dann in eine eigene Funktion auslagern die von beiden bisherigen Stellen aufgerufen wird.